frontend-web/webclient/app/ui-components/Input.tsx

```diff
@@ -160,17 +160,13 @@ const InputLabelClass = injectStyle("input-label", k => `
     }
     
     ${k}[data-left-label="true"] {
-        border-top-left-radius: 5px;
-        border-bottom-left-radius: 5px;
```

Comment:
These are used in `CreateProduct.tsx` and `DataList.tsx`. Is this a mistake?

Reply:
I don't see any places where both rightLabel and leftLabel are being used, except for the permissions-editor for applications, but either way I don't think the previous styling was correct. The border radius is already 5, and also this prevented adding both rightLabel and leftLabel at the same time, since they would overwrite each other's border-radius.

Reply:
I think you're correct. Marking as resolved.